src/frequenz/sdk/microgrid/component_data.py

```diff
@@ -52,7 +57,18 @@ def from_proto(cls, raw: microgrid_pb.ComponentData) -> ComponentData:
 
 @dataclass(frozen=True)
 class MeterData(ComponentData):
-    """A wrapper class for holding meter data."""
+    """A wrapper class for holding meter data.
+
+    Attributes:
+        active_power: the 3-phase active power, in Watts, represented in the passive
+            sign convention.
```

Comment:
This should also be:
            +ve power consumption, away from the grid.
            -ve power supply into the grid.

Reply:
Ok, I will add it (it is indeed documented like that in the protobuf message), but I was following the docstrings proposed above:
https://github.com/frequenz-floss/frequenz-sdk-python/pull/4#issuecomment-1267086625

Reply:
Can you create an issue in https://github.com/frequenz-floss/frequenz-api-microgrid (or even a PR)? If we see things that are wrong in the microgrid API, we should fix it!

Reply:
@leandro-lucarella-frequenz I think microgrid API is all good. The docstrings are complete there:
https://github.com/frequenz-floss/frequenz-api-microgrid/blob/v0.x.x/proto/frequenz/api/microgrid/common.proto#L214-L218

What Ela pointed out was only missing here. (except if we want to also add this part `represented in the passive sign convention.` to the docstring in microgrid-api, but I am not sure if it is needed.